tools/generate_hotkeys.py

```diff
@@ -38,6 +38,9 @@ def lint_hotkeys_file() -> None:
             various_key_combinations = various_key_combination(key_combinations_list)
             # Check description style
             if not re.match(help_text_style, help_text):
+                various_key_combinations = various_key_combinations.replace(
+                    "<kbd>", ""
+                ).replace("</kbd>", "")
```

Comment:
I think you undid whatever you did here?

Reply:
Oh black took effect. Why not just use `key_combinations_list` in the print statement instead of undoing whatever was done in the `various_key_combinations()` function.  (What does `" / ".join(key_combinations_list)` look like?)

Reply:
Also I don't like that fact that `various_key_combinations` is both a variable name and a function name.

Reply:
I have used ``" / ".join(key_combinations_list)`` in my recent commit, it shows key combinations like ``ctrl _`` instead of earlier ``ctrl + _`` and also for two key combinations it shows something like ``up / ctrl p``.

Reply:
I think it makes sense to show it this way as we are anyway asking them to edit config/keys.py (`ctlr _`) and not the markdown file (`ctrl + _`). The other one might be confusing if we show markdown and expect them to edit python.